runtime/src/main/java/io/quarkiverse/cxf/CXFRecorder.java

```diff
@@ -19,79 +20,22 @@
-                endpointAddress = "http://localhost:8080";
-            }
-            // default is sei name without package
-            if (relativePath == null) {
-                String serviceName = sei.toLowerCase();
-                if (serviceName.contains(".")) {
-                    serviceName = serviceName.substring(serviceName.lastIndexOf('.') + 1);
-                }
-                relativePath = "/" + serviceName;
-            }
-            if (!relativePath.equals("/") && !relativePath.equals("")) {
-                endpointAddress = endpointAddress.endsWith("/")
-                        ? endpointAddress.substring(0, endpointAddress.length() - 1)
-                        : endpointAddress;
-                endpointAddress = relativePath.startsWith("/") ? endpointAddress + relativePath
-                        : endpointAddress + "/" + relativePath;
-            }
-
-            CXFClientInfo cfg = new CXFClientInfo();
-            cfg.init(sei,
-                    endpointAddress,
-                    cxfEndPointConfig != null ? cxfEndPointConfig.wsdlPath.orElse(null) : null,
-                    soapBinding,
-                    wsNamespace,
-                    wsName,
-                    cxfEndPointConfig != null ? cxfEndPointConfig.endpointNamespace.orElse(null) : null,
-                    cxfEndPointConfig != null ? cxfEndPointConfig.endpointName.orElse(null) : null,
-                    cxfEndPointConfig != null ? cxfEndPointConfig.username.orElse(null) : null,
-                    cxfEndPointConfig != null ? cxfEndPointConfig.password.orElse(null) : null,
-                    classNames);
-            if (cxfEndPointConfig != null && cxfEndPointConfig.inInterceptors.isPresent()) {
-                cfg.getInInterceptors().addAll(cxfEndPointConfig.inInterceptors.get());
-            }
-            if (cxfEndPointConfig != null && cxfEndPointConfig.outInterceptors.isPresent()) {
-                cfg.getOutInterceptors().addAll(cxfEndPointConfig.outInterceptors.get());
-            }
-            if (cxfEndPointConfig != null && cxfEndPointConfig.outFaultInterceptors.isPresent()) {
-                cfg.getOutFaultInterceptors().addAll(cxfEndPointConfig.outFaultInterceptors.get());
-            }
-            if (cxfEndPointConfig != null && cxfEndPointConfig.inFaultInterceptors.isPresent()) {
-                cfg.getInFaultInterceptors().addAll(cxfEndPointConfig.inFaultInterceptors.get());
-            }
-            if (cxfEndPointConfig != null && cxfEndPointConfig.features.isPresent()) {
-                cfg.getFeatures().addAll(cxfEndPointConfig.features.get());
-            }
-            return cfg;
-        };
+    /**
+     * Create CXFClientInfo supplier.
+     * <p>
+     * This method is called once per @WebService *interface*. The idear is to produce a default client config for a
+     * given SEI.
+     */
+    public RuntimeValue<CXFClientInfo> cxfClientInfoSupplier(CXFClientData ws) {
```

Comment:
Kindly change `CXFClientData ws` to say `CXFClientData cxfClientData` as it improves readability and maintainability.

Reply:
Changed. Altough I believe that if there is jsut one parameter and a very short body, the variable name becomes irrelevant. All that matters is the variable's type ..

Reply:
Done.